scripts/test_schema_fallback.py

```
@@ -0,0 +1,43 @@
+import ifcopenshell
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)